test/fixture/main.tf

```diff
@@ -73,3 +73,15 @@ module "aks_without_monitor" {
   net_profile_pod_cidr           = "10.1.0.0/16"
   depends_on                     = [azurerm_resource_group.main]
 }
+
+module "aks_cluster_name" {
+  source                         = "../.."
+  cluster_name                   = "test-cluster"
+  prefix                         = "prefix"
+  resource_group_name            = azurerm_resource_group.main.name
+  enable_log_analytics_workspace = true
+  cluster_analytics_workspace    = "test-cluster"
```

Comment:
cluster_log_analytics_workspace_name = "test-cluster"

Reply:
I applied this change to the test.